packages/web-react/src/components/Tooltip/useFloating.ts

```diff
@@ -54,12 +58,27 @@ export const useFloating = (props: UseTooltipUIProps) => {
   } = props;
 
   const [maxWidth, setMaxWidth] = useState<number | undefined>(undefined);
+  const [isClicked, setIsClicked] = useState(false);
   const mainAxisOffset = cornerOffset + tooltipArrowWidth;
 
   // Floating UI library settings
   const { x, y, refs, context, placement, middlewareData } = useFloatingUI({
     open: isOpen,
-    onOpenChange: onToggle,
+    onOpenChange: (open, event, reason) => {
```

Comment:
What about using `onMouseEvent`/`oncCursorEvent` or simply `onOpen`. I am not satisfied with the naming. It does not seem right to me. I am not sure from the name what situations this handler handles.

Reply:
it is set by FloatingUI:
https://floating-ui.com/docs/useFloating#:~:text=%3F%3A%20boolean%3B-,onOpenChange,-%3F(